### ISSUE TITLE
Installation error: package 'ewceData' requires R >= 4.1

### ISSUE THREAD

**Issue by zhanglab2008** [NathanSkene/EWCE#31]
Hi,
Thanks for writing this great tools. However, I came across an error across all operation systems (Windows, Mac and Ubuntu 20.4) when I installed the package.
```{r }
ERROR: this R is version 4.0.4, package 'ewceData' requires R >= 4.1
Error: Failed to install 'ewceData' from GitHub:
  (converted from warning) installation of package ‘/tmp/Rtmp9xG1JD/file26f92ae904c2/ewceData_0.99.6.tar.gz’ had non-zero exit status
```
Could you suggest if this is a bug or if I miss something?
Many thanks!
Zhang

**Comment by NathanSkene** [comment]
Hi,

We're trying to get the package back onto bioconductor but their submission
system has required that we break the master branch. I suggest installing
an earlier commit from before 1 month ago. Would send a link but am on my
phone at the moment! Will reply shortly with the specific command to use.

**Comment by zhanglab2008** [comment]
That works! Thanks!

**Comment by madnessfish** [comment]
Hi there! I have also encountered the same problem as @zhanglab2008. Would you mind posting the specific command? Thank you so much!!

**Comment by numpde** [comment]
@madnessfish 
[Earlier version](https://github.com/NathanSkene/EWCE/tree/0aeef50) (from ~Jan 26, 2021):

```
devtools::install_github("nathanskene/ewce@0aeef50")
```

**Comment by Al-Murphy** [comment]
Hi both, the data has been removed from the git history of EWCE as per a requirement of Bioconductor. EWCE will be reinstated on Bioconductor in the next release (likely April/May this year). To make EWCE usable for those with R < 4.1 in the meantime, we have made a separate repository based on a previous commit of EWCE and added back in the necessary data. See [neurogenomics/EWCE](https://github.com/neurogenomics/EWCE) for details and feel free to create issues on that repo if anything is wrong.
